### ISSUE TITLE
Which teacher model I can choose in prompt distillation?

### ISSUE THREAD

**Issue by aunaun913** [thinking-machines-lab/tinker-cookbook#356]
If I only can choose those models in price sheet as teacher model?

**Comment by YujiaBao** [comment]
Good question! It depends on how you're using the teacher:

**If you're sampling from Tinker's API** (i.e., using a completer to generate teacher outputs), then yes, you can only choose from the models in our [model lineup](https://tinker-docs.thinkingmachines.ai/tinker/models/).

**If you have a different teacher model** that we don't host, you can still do distillation as long as the teacher and student share the same vocabulary set. In that case, you can supply the teacher's top-K logits yourself and use our [top-K distillation](https://tinker-docs.thinkingmachines.ai/tinker/losses/cross-entropy/) to train the student.
